### ISSUE TITLE
Does the preprocessing script work for 2.2?

### ISSUE THREAD

**Issue by GeekAlexis** [budzianowski/multiwoz#101]
There is no instruction in README on how to run preprocessing for 2.2. Instead, it implies that delexicalization is only compatible with earlier versions. Can someone confirm if that's the case? Thanks.

**Comment by XiaoxueZang** [comment]
Yes, it's correct. If you want the data format to be consistent with that of MultiWOZ 2.1, please try using [convert_to_multiwoz_format.py](https://github.com/budzianowski/multiwoz/blob/master/data/MultiWOZ_2.2/convert_to_multiwoz_format.py).
